lib/context-util.c

```diff
@@ -139,18 +139,18 @@ tcti_tabrmd_init (void)
 
     rc = tss2_tcti_tabrmd_init(NULL, &size);
     if (rc != TSS2_RC_SUCCESS) {
-        LOG_ERR ("Failed to get size for TABRMD TCTI context: 0x%" PRIx32, rc);
+        LOG_ERR ("Failed to get size for TABRMD TCTI context: 0x%" PRIx32 "\n", rc);
```

Comment:
The old code that the LOG_ macros replaced was often time missing new lines or adding superfluous formatting. The goal of the LOG_ macros was to provide consistent behavior of tool output that is not part of normal operation.

The more I think about this, we really don't need LOG_ macros for output, except to control quiet. The LOG_ macros were designed for optional/error output in a specific formatted version. I don't think general tool output is appropriate for them.

Maybe we could just place a macro in main.h TOOL_OUTPUT() that outputs to stdout and they control the formatting completely.